packages/zwave-js/src/lib/commandclass/SceneActuatorConfigurationCC.ts

```diff
@@ -0,0 +1,395 @@
+					throwUnsupportedPropertyKey(
+						this.ccId,
+						property,
+						propertyKey,
+					);
+				}
+				return (await this.get(propertyKey))?.[property];
+			}
+			default:
+				throwUnsupportedProperty(this.ccId, property);
+		}
+	};
+
+	public async set(
+		sceneId: number,
+		dimmingDuration: Duration,
+		level?: number,
+	): Promise<void> {
+		this.assertSupportsCommand(
+			SceneActuatorConfigurationCommand,
+			SceneActuatorConfigurationCommand.Set,
+		);
+
+		const cc = new SceneActuatorConfigurationCCSet(this.driver, {
+			nodeId: this.endpoint.nodeId,
+			endpoint: this.endpoint.index,
+			sceneId,
+			dimmingDuration,
+			level,
+		});
+
+		await this.driver.sendCommand(cc, this.commandOptions);
+	}
+
+	public async get(
+		sceneId: number,
+	): Promise<
+		| {
+				sceneId: number;
+				level: number;
+				dimmingDuration: Duration;
+		  }
+		| undefined
+	> {
+		this.assertSupportsCommand(
+			SceneActuatorConfigurationCommand,
+			SceneActuatorConfigurationCommand.Get,
+		);
+
+		const cc = new SceneActuatorConfigurationCCGet(this.driver, {
+			nodeId: this.endpoint.nodeId,
+			endpoint: this.endpoint.index,
+			sceneId: sceneId,
+		});
+		const response = await this.driver.sendCommand<SceneActuatorConfigurationCCReport>(
+			cc,
+			this.commandOptions,
+		);
+
+		if (response) {
+			return pick(response, ["sceneId", "level", "dimmingDuration"]);
+		}
+	}
+}
+
+@commandClass(CommandClasses["Scene Actuator Configuration"])
+@implementedVersion(1)
+export class SceneActuatorConfigurationCC extends CommandClass {
+	declare ccCommand: SceneActuatorConfigurationCommand;
+
+	// interview is omitted, because our only option would be
+	// to query all 255 possible sceneIds, which is a lot of traffic
+	// for little benefit
+}
+
+interface SceneActuatorConfigurationCCSetOptions extends CCCommandOptions {
+	sceneId: number;
+	dimmingDuration: Duration;
+	level?: number;
+}
+
+@CCCommand(SceneActuatorConfigurationCommand.Set)
+export class SceneActuatorConfigurationCCSet extends SceneActuatorConfigurationCC {
+	public constructor(
+		driver: Driver,
+		options:
+			| CommandClassDeserializationOptions
+			| SceneActuatorConfigurationCCSetOptions,
+	) {
+		super(driver, options);
+		if (gotDeserializationOptions(options)) {
+			// TODO: Deserialize payload
+			throw new ZWaveError(
+				`${this.constructor.name}: deserialization not implemented`,
+				ZWaveErrorCodes.Deserialization_NotImplemented,
+			);
+		} else {
+			if (options.sceneId < 1 || options.sceneId > 255) {
+				throw new ZWaveError(
+					`scene id ${options.sceneId} out of range [1..255]`,
+					ZWaveErrorCodes.Argument_Invalid,
+				);
+			}
+			this.sceneId = options.sceneId;
+			this.dimmingDuration = options.dimmingDuration;
+			if (options.level != undefined) {
+				if (options.level < 0 || options.level > 255) {
+					throw new ZWaveError(
+						`level ${options.level} out of range [1..255]`,
+						ZWaveErrorCodes.Argument_Invalid,
+					);
+				}
```

Comment:
This is unnecessary (and the error message is wrong). If you pass a value into a buffer, it is automatically `&`-ed with `0xff`.